scripts/linux/publishReleasePackages.sh

```diff
@@ -166,3 +182,140 @@ if [[ $uploaded == false ]]; then
 else
     echo "Packages Uploaded"
 fi
+
+}
+
+publish_to_github()
+{   
+    #Investigate if this can be derived from a commit, Hardcode for now.
+    if [[ -z $BRANCH_NAME ]]; then
+        echo "No Branch Name Provided"
+        exit 1
+    fi
+    
+    branch_name=${BRANCH_NAME/"refs/heads/"/""}
+    echo "Branch Name is $branch_name"
+    
+     # Get the latest release from a given branch
+    echo "Fetch the latest release: "
+    url="https://api.github.com/repos/Azure/iotedge/releases"
+    header_content="Accept:application/vnd.github.v3+json"
+    header_auth="Authorization:token $GITHUB_PAT"
```

Comment:
Do we have a check somewhere in this script that `GITHUB_PAT` is defined?

Reply:
Yes, In Line 309